lib/produce/config.rb

```diff
@@ -84,5 +86,9 @@ def is_valid_language? language
     def skip_itc? value
       %w( true t 1 yes y ).include? value.to_s.downcase
     end
+
+    def skip_devcenter? value
```

Comment:
Don't like how this repeats `skip_itc?` method. Please rename `skip_itc?` to something like `is_truthy?`, and use it instead of both `skip_itc?` and `skip_devcenter?`.

To @KrauseFx: Ideally this should go to `fastlane_core`.